rate.go

```diff
@@ -91,6 +92,34 @@ func (l *Limiter) AllowRate(name string, rateLimit timerate.Limit) (delay time.D
 	return delay, allow
 }
 
+// Verify reports whether an event with given name may happen at time now.
+// It allows up to maxn events within duration dur.
+// The difference for the Allow method is that this method does not increment usage.
+func (l *Limiter) Verify(name string, maxn int64, dur time.Duration) (count, reset int64, allow bool) {
```

Comment:
This package is designed after https://godoc.org/golang.org/x/time/rate and in that package I see method `AllowN`. So what do you think about replacing `Verify` with `AllowN(name string, maxn int64, dur time.Duration, n int64)`? Then you can use `AllowN("hello", 100, time.Minute, 0)` to get current rate limit.